src/sendAPN.js

```diff
@@ -23,7 +23,7 @@ class APN {
   sendAPN(regIds, data) {
     const message = new apn.Notification({
       retryLimit: data.retries || -1,
-      expiry: data.expiry || defaultExpiry(data.timeToLive),
+      expiry: data.timeToLive || defaultExpiry(data.expiry),
```

Comment:
if `timeToLive` is set to `0` it will be evaluated as falsy and the default expiry will be calculated. this change does not achieve what you are trying to do if I am not mistaken